test/unit/tcache_max.c

```diff
@@ -189,6 +194,7 @@ TEST_BEGIN(test_tcache_max) {
 	test_skip_if(opt_prof);
 	test_skip_if(san_uaf_detection_enabled());
 
+	tcache_event = &tcache_event_no_ops;
```

Comment:
For safety, please change `tcache_event` back to its original value at the end of the test. Same below.